app/migration/mappers/training_training_programme_mapper.rb

```diff
@@ -1,5 +1,5 @@
 module Mappers
-  class TrainingProgrammeTypeMapper
+  class TrainingTrainingProgrammeMapper
```

Comment:
```suggestion
  class TrainingProgrammeMapper
```

Reply:
Good catch. It seems this happened in a few more files, so I pushed a commit to fix them all.